### DISCUSSION TITLE
Is it possible to plot real-time serial data by using Atoti?

### DISCUSSION THREAD

**Discussion by baversaglam** [atoti/atoti#808] [Q&A] [answered]
I am trying to plot my Arduino's serial data output on python. I want to use Atoti library for this, is it possible to plot my serial data in real-time? Which steps should I follow to do that? 

Thanks

**Comment by tibdex** [comment, marked as answer]
Yes it's possible. You could follow an approach similar to [this](https://docs.atoti.io/0.8.4/how_tos/watch_local_files.html) or [that](https://github.com/atoti/project-template/blob/2056078e9c5d23b797017b451248fef8f5ce42d5/app/app.py#L20-L23).

**Reply by baversaglam** [reply]
thanks!
